src/actions/vstudio/_vstudio.lua

```diff
@@ -340,7 +340,13 @@
 		-- Then the system libraries, which come undecorated
 		local system = config.getlinks(cfg, "system", "fullpath")
 		for i = 1, #system do
-			table.insert(links, path.appendextension(system[i], ".lib"))
+			-- Add extension if required
+			local link = system[i]
+			if link:match("[^.]+$") == link then
```

Comment:
It doesn't look like this regex would work if there's more than one `.` in the filename? Libs often have a version number or whatever in them too.